### PR TITLE
Added ;;cooldog command and some aliases

### PR DESCRIPTION
**Pull request by RealKC**
;;cooldog has the aliases ;;dogmeme and ;;dog
added ;;1ee7 alias to ;;leet

### CODE REVIEW

**Comment by Frederikam** [comment]
Please create a separate class for this. Preferably overriding `TextCommand`. This looks like a mess to have in the main class.

**Comment by RealKC** [comment]
OK, I assume it will have to import the same things  TextCommand.java does.

**Comment by Frederikam** [comment]
Mostly, but it needs to `extends TextCommand`. I recommend getting an IDE to handle the import statements.

File: src/main/java/fredboat/command/fun/DogCommand.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
++ "┈┈╲╲┈┈┈┈┈▏┈▏┈▔▔▔▆ ------- BORF!\n"
++ "┈┈┈╲▔▔▔▔▔╲╱┈╰┳┳┳╯ \n"
++ "╱╲╱╲▏┈┈┈┈┈┈▕▔╰━╯ \n"
++ "▔╲╲╱╱▔╱▔▔╲╲╲╲┈┈┈ \n"
++ "┈┈╲╱╲╱┈┈┈┈╲╲▂╲▂┈ \n"
++ "┈┈┈┈┈┈┈┈┈┈┈╲╱╲╱┈\n"
++ "Spread him to take over Discord!(Don't spam him though!)";
+//dog created by @Ace#8925
+
+        public TextCommand() {
```
Comment (Frederikam) [inline src/main/java/fredboat/command/fun/DogCommand.java]: Should be `DogCommand` instead of `TextCommand`

**Comment by RealKC** [comment]
Idk why Travis has problems.

**Pull request merged**